go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/lib/pq v1.10.6
 	github.com/minio/minio-go/v7 v7.0.45
 	github.com/mwitkow/go-proto-validators v0.3.2
-	github.com/percona-platform/dbaas-api v0.0.0-20221124165531-bd24900964a7
```

Comment:
This dep wasn't switched to `main` before merging https://github.com/percona/pmm/pull/1391